test/fluree/db/ledger/query/analytical_test.clj

```diff
@@ -0,0 +1,160 @@
+(ns fluree.db.ledger.query.analytical-test
+  (:require [clojure.test :refer :all]
+            [fluree.db.test-helpers :as test]
+            [fluree.db.api :as fdb]
+            [fluree.db.util.async :refer [<??]]))
+
+(use-fixtures :once test/test-system)
+
+(deftest analytical-with-prefix-two-tuple-subject
+  (testing "Analytical query with prefix, with two-tuple subject"
+    (let [crawl-query {:select "?nums"
+                       :where  [["$fdb", ["person/handle", "zsmith"], "person/favNums", "?nums"]]}
+          ledger      (test/rand-ledger test/ledger-chat
+                                        {:http/schema ["chat.edn" "chat-preds.edn"]})
+          {:keys [block]} (test/transact-data ledger "chat.edn")
+          db          (fdb/db (:conn test/system) ledger {:syncTo block})
+          res         (<?? (fdb/query-async db crawl-query))]
+      (is (= #{5 645 28 -1 1223} (set res))
+          (str "Query result was: " (pr-str res))))))
```

Comment:
I could certainly make it clearer but I have the same intent as your understanding. The idea is "this test failed, and it's expectations are around some query results, so here's that query result so you can debug it."

Let me know if you have a suggestion to better align my execution with that intent.

Reply:
gotcha. this totally makes sense and is a very useful idea. maybe adding something like "Unexpected result from query. Query result was ..." to make it a little clearer,. but now that i've seen this pattern and you've explained it to me, i'm stealing it.

Reply:
Yeah, makes sense. The only reason it's as terse as it is is because when it shows up it's already surrounded by the test framework's messages about a test failure. But I like adding "unexpected" to it, so I'll do that.